docker-compose.yml

```diff
@@ -61,11 +34,10 @@ slave2:
     - MESOS_ISOLATOR=cgroups/cpu,cgroups/mem
     - MESOS_LOG_DIR=/var/log
     - MESOS_RESOURCES=ports(*):[31000-31100]
-    - MESOS_MODULES=file:///isolator/build/isolator/modules.json
+    - MESOS_MODULES=file:///calico/modules.json
     - MESOS_ISOLATION=com_mesosphere_mesos_NetworkIsolator
     - MESOS_HOOKS=com_mesosphere_mesos_NetworkHook
     - ETCD_AUTHORITY=etcd:4001
-    - MESOS_ATTRIBUTES=public:true
```

Comment:
@ConnorDoyle 
There doesn't actually need to be a requirement that the nexthop route added via `add-container-route.sh` belongs to the Slave which the UI is running on. Since each Slave is also BGP speaker, even if the app ends up on Slave2, Slave1 will be able to nexthop the HTTP request to Slave2.

I've confirmed this by swapping the route added by `add-container-route.sh` to the other Slave and the demo still loads fine.